js_modules/dagster-ui/packages/ui-core/src/runs/QueuedRunCriteriaDialog.tsx

```diff
@@ -172,31 +177,27 @@ const QueuedRunCriteriaDialogContent = ({run}: ContentProps) => {
             </td>
           </tr>
         ) : null}
-        {runIsOpConcurrencyLimited ? (
+        {poolOpGranularityRunLimited || poolRunGranularityRunLimited ? (
           <tr>
             <td>
               <Box flex={{direction: 'row', alignItems: 'center', gap: 4}}>
-                <div>Initial concurrency keys</div>
+                <div>Pools</div>
                 <Tooltip
                   placement="bottom"
-                  content="Op/asset concurrency limits are set on all of the initial steps in this run. This run will not start until there are available slots for at least one step."
+                  content={
```

Comment:
Might be nice to have a wrapper div on this with a specified width -- the tooltip would look a little nicer if it wrapped onto multiple lines.